graphql-spqr-spring-boot-autoconfigure/src/main/java/io/leangen/graphql/spqr/spring/web/GraphQLController.java

```diff
@@ -77,7 +77,7 @@ public Object executeFormPost(@RequestParam Map<String, String> queryParams,
     @GetMapping(
             value = "${graphql.spqr.http.endpoint:/graphql}",
             produces = MediaType.APPLICATION_JSON_UTF8_VALUE,
-            headers = "Connection!=Upgrade"
+            headers = { "Connection!=Upgrade", "Connection!=keep-alive, Upgrade" }
```

Comment:
Wouldn't this code be better?
`
headers = { "Connection!=*Upgrade*" }
`
to support `Upgrade, keep-alive` too, for example.